strider/server.py

```diff
@@ -412,21 +419,18 @@ async def extract_results(query_id, since, limit, offset, database):
     ]
 
 
-@APP.post('/plan', response_model=list[dict])
+@APP.post('/plan', response_model=list[list[str]])
 async def generate_traversal_plan(
         query: Query,
-) -> list[dict]:
+) -> list[list[str]]:
     """Generate plans for traversing knowledge providers."""
     query_graph = query.message.query_graph.dict()
 
-    await fill_categories_predicates(query_graph, logging.getLogger())
-    standardize_graph_lists(query_graph)
-    plans = await generate_plans(query_graph)
+    # await fill_categories_predicates(query_graph, logging.getLogger())
+    # standardize_graph_lists(query_graph)
+    plans, kps = await generate_plans(query_graph)
```

Comment:
`kps` is unused, and commented code could be removed.

Reply:
7c19566a15dd8ac747aacb39a870f9b02b7425c1